src/data_source/storage/sql.rs

```diff
@@ -364,6 +360,16 @@ impl<DB: Database> Config<DB> {
         self.pool_opt = self.pool_opt.max_connections(max);
         self
     }
+
+    /// Log at WARN level any time a SQL statement takes longer than `threshold`.
+    ///
+    /// The default threshold is 1s.
+    pub fn slow_statement_threshold(mut self, threshold: Duration) -> Self {
```

Comment:
do we want to set this at sequencer side through an env variable?

Reply:
probably yeah